src/components/UserAvatar.js

```diff
@@ -0,0 +1,13 @@
+export default class UserAvatar {
```

Comment:
`UserAvatar` is not needed because `UserInfo` should set/get the profile data (including the **avatar**)